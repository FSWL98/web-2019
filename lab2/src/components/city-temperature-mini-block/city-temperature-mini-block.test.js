import React from "react";
import CityTemperatureMiniBlock from "./city-temperature-mini-block";
import renderer from 'react-test-renderer';

const cityinfo = {
    icon: "13d",
    name: "Moscow",
    temperature: "10"
};

it("Usual temperature-name mini block", () => {
    const component = renderer.create(<CityTemperatureMiniBlock cityinfo={cityinfo} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
