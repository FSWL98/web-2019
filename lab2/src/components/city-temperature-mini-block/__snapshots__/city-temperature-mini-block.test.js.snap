// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Usual temperature-name mini block 1`] = `
<div
  className="m-0 d-flex justify-content-between align-items-center row"
>
  <div
    className="d-flex pl-0 align-items-center col-md-9"
  >
    <p
      className="city-temperature-mini-block__city-name mr-5 mb-0"
    >
      Moscow
    </p>
    <span
      className="city-temperature-mini-block__temperature mr-1"
    >
      10
      Â°C
    </span>
    <img
      alt="icon"
      height="45"
      src="https://openweathermap.org/img/wn/13d.png"
    />
  </div>
  <div
    className="col"
  />
</div>
`;
